src/js/containers/search/filters/budgetCategory/BudgetCategorySearchContainer.jsx

```diff
@@ -18,7 +18,26 @@ const propTypes = {
     updateSelectedObjectClasses: React.PropTypes.func
 };
 
+
+const ga = require('react-ga');
+
 export class BudgetCategorySearchContainer extends React.Component {
+    static logBudgetFunctionFilterEvent(type) {
+        ga.event({
+            category: 'Search Page Filter Applied',
+            action: `Applied Budget Function Filter`,
```

Comment:
Combined the two budget category events and changed this to a dynamic string with an `event` parameter for Budget Function/ Federal Account